decidim/decidim
# Add Meetings' attendees metric

#### :tophat: What? Why?
This PR adds a new metric "Meetings attendees".

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to https://meta.decidim.org/processes/roadmap/f/122/proposals/16398
- Fixes #?

#### Testing
1. Green pipeline 
2. Compute metrics 
3. Create a meeting and close it with some attendees count 
4. Rebuild metrics
5. See Metric appearing in Space metrics

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![Description](URL)

:hearts: Thank you!
